jitpack.yml

```diff
@@ -0,0 +1,6 @@
+jdk:
+  - openjdk17
+before_install:
+  - echo "Hello jitpack.yml before_install"
```

Comment:
This file seems to not be picked up at all; I don't see this being printed to the logs